### PR TITLE
Barbara's changes to the Our Community Block

### PR DESCRIPTION
**Pull request by buttonwillowsix**
This is a couple of simple text edits, two new images, and two images swapped. Also, haven't done a pull request in ages, so hoping this is right!

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,5 +25,12 @@
   "description": "=========================== How to Use Grunt with Stanford Module Example ===========================",
   "main": "Gruntfile.js",
   "author": "",
-  "license": "MIT"
+  "license": "MIT",
+  "devDependencies": {
```
Comment (sherakama) [inline package.json]: Hey Dena,

Grunt and the other libraries are already listed as a dependency. They don't also need to be in the devDependencies for this.

File: modules/stanford_earth_mosaic_blocks/src/Plugin/Block/MosaicBlockDefault.php [hunk 2]
```diff
@@ -40,7 +40,7 @@ public function build() {
               '#uri' => drupal_get_path('module', 'stanford_earth_mosaic_blocks') . '/img/studentmain.png',
             ],
             'is_right' => TRUE,
-            'quote' => $this->t('We face enormous hrdles as a society in dealing with a changing climate, over extraction of resources, and biodiversity loss. Still, I remain optimistic in our ability to identify and design solutions.'),
```
Comment (sherakama) [inline modules/stanford_earth_mosaic_blocks/src/Plugin/Block/MosaicBlockDefault.php:43]: Ooops. Thanks for catching that.

**Review by sherakama — CHANGES_REQUESTED** [review]
Most of this looks good. Please remove the devDependencies. Also, is there a Jira ticket associated with this?

**Comment by sherakama** [comment]
@buttonwillowsix

**Comment by buttonwillowsix** [comment]
Oops! Sorry about those devDependencies. Will remove. I think that happened
when I was messing around with my local environment.

There is a Jira ticket associated with this, but I had no idea about how to
link them up!


*Dena DeBry*
*Buttonwillow•Six <http://www.buttonwillowsix.com>* LLC
dena@buttonwillowsix.com
<http://www.buttonwillowsix.com>
*(801) 252-6006*
*Skype:* denanatalie

**Comment by sherakama** [comment]
Our Jira and GitHub organizations can talk to each other if you use the smart commit syntax in your commit message. It goes:

(ticket id)(colon) (commit message)(ending punctuation) 
eg:
EARTH-432: Added some stuff.

Please remove the dev dependencies and I can merge.

**Comment by buttonwillowsix** [comment]
@sherakama, Added two more commits  (one for removing the devDependencies and the other because I realized the alt tags were not updated.)

**Review by sherakama — APPROVED** [review]

**Pull request merged**